src/View/Helper/CrudViewHelper.php

```diff
@@ -323,6 +329,16 @@ public function associations()
     }
 
     /**
+     * Returns the main table dispaly field column
+     *
+     * @return void
+     */
+    public function displayField()
```

Comment:
It's this discrepancy which lead me to ask:

    if ($field === $this->displayField()) {
                          ^ explicit accessor
            $value = ... $data->get($this->getViewVar('primaryKey'))]);
                                           ^ no explicit accessor

Would be best to be consistent, but creating getters to wrap getViewVar seems a bit derp